src/coord.jl

```diff
@@ -1,17 +1,81 @@
-"""
-    Coord <: FieldVector{4, Float64}
+const Coord = SVector{4, Float64}
```

Comment:
Yeah this probably makes sense. What drew you to this conclusion?

Reply:
Mostly it felt wrong that the field names were not necessarily correct, for instance x and y could be reversed. I wrote a bit more detail in the commit message: https://github.com/JuliaGeo/Proj4.jl/pull/51/commits/1d0c03ef93701a2f4a2d36d8ea3eea0238e152d1

Reply:
Ah yes I forgot about CRS-dependent axis order. That's a very good reason to go with indices only.